src/main/java/ch/admin/bar/siardsuite/presenter/archive/browser/ArchiveBrowserPresenter.java

```diff
@@ -0,0 +1,227 @@
+package ch.admin.bar.siardsuite.presenter.archive.browser;
+
+import ch.admin.bar.siardsuite.Controller;
+import ch.admin.bar.siardsuite.component.TreeBuilder;
+import ch.admin.bar.siardsuite.component.ButtonBox;
+import ch.admin.bar.siardsuite.component.IconButton;
+import ch.admin.bar.siardsuite.component.TwoStatesButton;
+import ch.admin.bar.siardsuite.component.rendering.TreeItemsExplorer;
+import ch.admin.bar.siardsuite.model.TreeAttributeWrapper;
+import ch.admin.bar.siardsuite.presenter.StepperPresenter;
+import ch.admin.bar.siardsuite.presenter.tree.DetailsPresenter;
+import ch.admin.bar.siardsuite.util.CastHelper;
+import ch.admin.bar.siardsuite.util.I18n;
+import ch.admin.bar.siardsuite.util.fxml.FXMLLoadHelper;
+import ch.admin.bar.siardsuite.view.RootStage;
+import io.github.palexdev.materialfx.controls.MFXButton;
+import io.github.palexdev.materialfx.controls.MFXStepper;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.MultipleSelectionModel;
+import javafx.scene.control.TreeItem;
+import javafx.scene.control.TreeView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.StackPane;
+import lombok.val;
+
+import static ch.admin.bar.siardsuite.util.I18n.bind;
+import static ch.admin.bar.siardsuite.util.OptionalHelper.ifPresentOrElse;
+
+/**
+ * Presentes an archive - either when archiving a database (always only metadata) or when a SIARD Archive
+ * file was opened to browse the archive content
+ */
+public class ArchiveBrowserPresenter extends StepperPresenter {
+
+    @FXML
+    protected TreeView<TreeAttributeWrapper> treeView;
+    @FXML
+    protected IconButton saveChangesButton;
+    @FXML
+    protected IconButton dropChangesButton;
+    @FXML
+    protected TwoStatesButton tableSearchButton;
+    @FXML
+    protected MFXButton metaSearchButton;
+    @FXML
+    protected AnchorPane contentPane;
+    @FXML
+    public Label titleTableContainer;
+    @FXML
+    public Label errorMessageLabel;
+    @FXML
+    public StackPane rightTableBox;
+    @FXML
+    private ButtonBox buttonBox;
```

Comment:
the buttonBox seems to be unused. Please verify and remove the property

Reply:
Done. The whole view is no longer used since the changes in this PR.